server/s3_bucket_metadata_v1.cc

```diff
@@ -96,15 +84,26 @@ void S3BucketMetadataV1::set_state(S3BucketMetadataState state) {
   this->state = state;
 }
 
+S3GlobalBucketIndexMetadata*
+S3BucketMetadataV1::get_global_bucket_index_metadata() {
+  if (!global_bucket_index_metadata) {
+    global_bucket_index_metadata =
+        global_bucket_index_metadata_factory
+            ->create_s3_global_bucket_index_metadata(request, bucket_name,
+                                                     account_id, account_name);
+  }
+  return global_bucket_index_metadata.get();
+}
+
 // Load {B1, A1} in global_bucket_list_index_oid, followed by {A1/B1, md} in
 // bucket_metadata_list_index_oid
-void S3BucketMetadataV1::load(std::function<void(void)> on_success,
-                              std::function<void(void)> on_failed) {
+void S3BucketMetadataV1::load(const S3BucketMetadata& src,
+                              CallbackHandlerType callback) {
```

Comment:
can you rename callback variable to indicate actual work done by this call back, like metaLoadCallback

Reply:
Done